kubernetes/client/apis/custom_objects_api.py

```diff
@@ -672,6 +672,240 @@ def get_cluster_custom_object_with_http_info(self, group, version, plural, name,
+            raise ValueError("Missing the required parameter `group` when calling `get_cluster_custom_object_scale`")
+        # verify the required parameter 'version' is set
+        if ('version' not in params) or (params['version'] is None):
+            raise ValueError("Missing the required parameter `version` when calling `get_cluster_custom_object_scale`")
+        # verify the required parameter 'plural' is set
+        if ('plural' not in params) or (params['plural'] is None):
+            raise ValueError("Missing the required parameter `plural` when calling `get_cluster_custom_object_scale`")
+        # verify the required parameter 'name' is set
+        if ('name' not in params) or (params['name'] is None):
+            raise ValueError("Missing the required parameter `name` when calling `get_cluster_custom_object_scale`")
+
+
+        collection_formats = {}
+
+        path_params = {}
+        if 'group' in params:
+            path_params['group'] = params['group']
+        if 'version' in params:
+            path_params['version'] = params['version']
+        if 'plural' in params:
+            path_params['plural'] = params['plural']
+        if 'name' in params:
+            path_params['name'] = params['name']
+
+        query_params = []
+
+        header_params = {}
+
+        form_params = []
+        local_var_files = {}
+
+        body_params = None
+        # HTTP header `Accept`
+        header_params['Accept'] = self.api_client.\
+            select_header_accept(['application/json', 'application/yaml', 'application/vnd.kubernetes.protobuf'])
+
+        # HTTP header `Content-Type`
+        header_params['Content-Type'] = self.api_client.\
+            select_header_content_type(['*/*'])
+
+        # Authentication setting
+        auth_settings = ['BearerToken']
+
+        return self.api_client.call_api('/apis/{group}/{version}/{plural}/{name}/scale', 'GET',
+                                        path_params,
+                                        query_params,
+                                        header_params,
+                                        body=body_params,
+                                        post_params=form_params,
+                                        files=local_var_files,
+                                        response_type='object',
+                                        auth_settings=auth_settings,
+                                        async_req=params.get('async_req'),
+                                        _return_http_data_only=params.get('_return_http_data_only'),
+                                        _preload_content=params.get('_preload_content', True),
+                                        _request_timeout=params.get('_request_timeout'),
+                                        collection_formats=collection_formats)
+
+    def get_cluster_custom_object_status(self, group, version, plural, name, **kwargs):
```

Comment:
I checked out to this branch and I tested this method.
this method returns 404 for an existing object,his sister (`get_namespaced_custom_object`)  with exactly the same parameters, returns the object successfully.

Reply:
I need this method :-S
https://github.com/kubernetes-client/python/issues/607

Reply:
which version of kubernetes are you using? custom object subresources are enabled by default in kubernetes 1.11, in 1.10 you have to enable it with a feature gate.
You might also need to define the status in the custom resource definition, but I'm not sure about that, see https://github.com/kubernetes/sample-controller/blob/master/artifacts/examples/crd-status-subresource.yaml